config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet

```diff
@@ -8,13 +8,13 @@
             alert: 'Prow images are stale',
             expr: |||
               time()-max(prow_version) > %d * 24 * 3600
-            ||| % $._config.prowImageStaleByDays,
-            'for': '5m',
+            ||| % $._config.prowImageStaleByDays.daysStale,
+            'for': '%s' % $._config.prowImageStaleByDays.hoursToTrigger,
```

Comment:
Nit: No formatting is needed here, you can use the variable directly.

Reply:
Done